src/models/workflows.rs

```diff
@@ -93,3 +93,11 @@ pub struct Step {
     #[serde(skip_serializing_if = "Option::is_none")]
     pub completed_at: Option<chrono::DateTime<chrono::Utc>>,
 }
+
+#[derive(Debug, Clone, PartialEq, Serialize, Deserialize, Default)]
+#[non_exhaustive]
+pub struct WorkflowDispatch {
+    #[serde(rename = "ref")]
+    pub ref_field: String,
```

Comment:
In general and in `octocrab`, I prefer to just use `r#` syntax rather than using `rename`. In my opinion it's easier and more to remember to add `r#`, rather than remember what this library called the field, even if it might less visually appealing. Especially when users are likely looking at the JSON or GitHub's API docs rather than octocrab's.
```suggestion
    pub r#ref: String,
```